README.md

```diff
@@ -1,52 +1,60 @@
-[![License](https://img.shields.io/github/license/ps3dev/ps3toolchain.svg)](./LICENSE)
+# PS3 Toolchain [![License](https://img.shields.io/github/license/ps3dev/ps3toolchain.svg)](./LICENSE)
 
-  What does this do?
- ====================
+## What does this do?
+- This program will automatically build and install a compiler and other
+tools used in the creation of homebrew software for the Sony PlayStation 3
+videogame system.
 
-  This program will automatically build and install a compiler and other
-  tools used in the creation of homebrew software for the Sony PlayStation 3
-  videogame system.
+## How do I use it?
 
-  How do I use it?
- ==================
+1. If you are using Linux or macOS, add this to the login script (`~/.bashrc` on Linux)
```

Comment:
And if not?  This wording seems to suggest that nothing needs to be done in that case.  While the old wording might not have given you the correct details for every such case, it did at least convey that you had to set the variables.

I like the wording in your "2." better, which tells you what needs to be done and then notes that the concrete example may be relevant only to Linux.  So maybe we can say something like:

> Set the PS3DEV and PS1LIGHT environment variables, and the executable path (Linux `~/.bashrc` example)

?